### PR TITLE
[INLONG-7636][Docker] Modify the Kafka config for Audit docker deployment

### LINKED ISSUE #7636: [Feature][Docker] Modify the Kafka config for Audit docker deployment
### Description

Modify the Kafka config for Audit docker deployment.

### Use case

_No response_

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by haifxu**
### Prepare a Pull Request

- Fixes #7636 

### Motivation

Modify the Kafka config for Audit docker deployment without MQ config.

### Modifications

Modify the Kafka config for Audit docker deployment

### CODE REVIEW

File: inlong-audit/audit-docker/audit-docker.sh [hunk 1]
```diff
@@ -25,9 +25,10 @@ sql_file="${file_path}"/sql/apache_inlong_audit.sql
 sql_ck_file="${file_path}"/sql/apache_inlong_audit_clickhouse.sql
 
 # replace the configuration for audit proxy
+sed -i "s/manager.hosts=.*$/manager.hosts=${MANAGER_OPENAPI_IP}:${MANAGER_OPENAPI_PORT}/g" "${store_conf_file}"
+sed -i "s/proxy.cluster.tag=.*$/proxy.cluster.tag=${CLUSTER_TAG}/g" "${store_conf_file}"
 if [ "${MQ_TYPE}" = "pulsar" ]; then
   sed -i "s/audit.config.proxy.type=.*$/audit.config.proxy.type=pulsar"/g "${store_conf_file}"
-  sed -i "s/audit.pulsar.server.url=.*$/audit.pulsar.server.url=pulsar:\/\/${PULSAR_BROKER_LIST}/g" "${store_conf_file}"
   sed -i "s/audit.pulsar.topic = .*$/audit.pulsar.topic = ${PULSAR_AUDIT_TOPIC}/g" "${store_conf_file}"
   sed -i "s/agent1.sinks.pulsar-sink-msg1.pulsar_server_url = .*$/agent1.sinks.pulsar-sink-msg1.pulsar_server_url = pulsar:\/\/${PULSAR_BROKER_LIST}/g" "${proxy_conf_file}"
```
Comment (dockerzhang) [inline inlong-audit/audit-docker/audit-docker.sh:33]: here need to remove too?

File: inlong-audit/audit-docker/audit-docker.sh [hunk 2]
```diff
@@ -25,9 +25,10 @@ sql_file="${file_path}"/sql/apache_inlong_audit.sql
...
   sed -i "s/agent1.sinks.pulsar-sink-msg2.pulsar_server_url = .*$/agent1.sinks.pulsar-sink-msg2.pulsar_server_url = pulsar:\/\/${PULSAR_BROKER_LIST}/g" "${proxy_conf_file}"
```
Comment (dockerzhang) [inline inlong-audit/audit-docker/audit-docker.sh:34]: ditto

File: inlong-audit/audit-docker/audit-docker.sh [hunk 3]
```diff
@@ -36,13 +37,11 @@ if [ "${MQ_TYPE}" = "pulsar" ]; then
 fi
 if [ "${MQ_TYPE}" = "kafka" ]; then
   sed -i "s/audit.config.proxy.type=.*$/audit.config.proxy.type=kafka"/g "${store_conf_file}"
-  sed -i "s/audit.kafka.server.url=.*$/audit.kafka.server.url=${KAFKA_SERVER_LIST}"/g "${store_conf_file}"
   sed -i "s/agent1.sinks.kafka-sink-msg1.bootstrap_servers = .*$/agent1.sinks.kafka-sink-msg1.bootstrap_servers = ${KAFKA_SERVER_LIST}/g" "${proxy_conf_file}"
```
Comment (dockerzhang) [inline inlong-audit/audit-docker/audit-docker.sh:40]: ditto

File: inlong-audit/audit-docker/audit-docker.sh [hunk 4]
```diff
@@ -36,13 +37,11 @@ if [ "${MQ_TYPE}" = "pulsar" ]; then
...
   sed -i "s/agent1.sinks.kafka-sink-msg2.bootstrap_servers = .*$/agent1.sinks.kafka-sink-msg2.bootstrap_servers = ${KAFKA_SERVER_LIST}/g" "${proxy_conf_file}"
```
Comment (dockerzhang) [inline inlong-audit/audit-docker/audit-docker.sh:41]: ditto

File: inlong-audit/audit-docker/audit-docker.sh [hunk 5]
```diff
@@ -36,13 +37,11 @@ if [ "${MQ_TYPE}" = "pulsar" ]; then
...
 fi
 if [ "${MQ_TYPE}" = "tubemq" ]; then
   sed -i "s/audit.config.proxy.type=.*$/audit.config.proxy.type=tube"/g "${store_conf_file}"
-  sed -i "s/audit.tube.masterlist=.*$/audit.tube.masterlist=${TUBE_MASTER_LIST}/g" "${store_conf_file}"
   sed -i "s/audit.tube.topic = .*$/audit.tube.topic = ${TUBE_AUDIT_TOPIC}/g" "${store_conf_file}"
   sed -i "s/agent1.sinks.tube-sink-msg1.master-host-port-list = .*$/agent1.sinks.tube-sink-msg1.master-host-port-list = ${TUBE_MASTER_LIST}/g" "${proxy_conf_file}"
   sed -i "s/agent1.sinks.tube-sink-msg2.master-host-port-list = .*$/agent1.sinks.tube-sink-msg2.master-host-port-list = ${TUBE_MASTER_LIST}/g" "${proxy_conf_file}"
```
Comment (dockerzhang) [inline inlong-audit/audit-docker/audit-docker.sh:47]: ditto

File: inlong-audit/audit-docker/audit-docker.sh (hunk 1 above)
Comment (haifxu) [reply inlong-audit/audit-docker/audit-docker.sh]: Done.

**Review by dockerzhang — APPROVED** [review]

**Comment by dockerzhang** [comment]
https://github.com/apache/inlong/pull/7571

the standalone mode needs to change, you could create another PR.

**Review by healchow — APPROVED** [review]

**Pull request merged**